cairo1-run/src/main.rs

```diff
@@ -61,6 +61,9 @@ struct Args {
     args: FuncArgs,
     #[clap(long = "print_output", value_parser)]
     print_output: bool,
+    // Append the returned values of main to to Output Segment
+    #[clap(long = "add_main_output", value_parser, conflicts_with = "proof_mode")]
```

Comment:
Why mutually exclusive? Isn't adding the program output to the output segment needed for proving?